Samples/outlook-set-signature/teamsapp.yml

```diff
@@ -0,0 +1,44 @@
+# yaml-language-server: $schema=https://developer.microsoft.com/json-schemas/teams-toolkit/teamsapp-yaml/1.0.0/yaml.schema.json
```

Comment:
We now use this link to reference yaml schema https://aka.ms/teams-toolkit/1.0.0/yaml.schema.json to reference yaml schema.
This is the link to the PR https://github.com/OfficeDev/TeamsFx/pull/8554/files

Reply:
fixed.